testsuite/MDAnalysisTests/core/test_groups.py

```diff
@@ -1461,3 +1461,151 @@ def test_decorator(self, compound, pbc, unwrap):
                 self.dummy_funtion(compound=compound, pbc=pbc, unwrap=unwrap)
         else:
             assert_equal(self.dummy_funtion(compound=compound, pbc=pbc, unwrap=unwrap), 0)
+
+
+@pytest.fixture()
+def tpr():
+    return mda.Universe(TPR)
```

Comment:
This will emit a warning and we emit quite a lot already. It may be worth muting the warning for that line:

```python
import warnings

@pytest.fixture()
def tpr():
     with warnings.catch_warnings():
         warnings.filterwarnings("ignore", message="No coordinate reader found")
         return mda.Universe(TPR)
```